deploy.ci.acceptance.mariadb.yml

```diff
@@ -116,10 +111,10 @@ services:
     search:
         engine: elastic
         version: '7.10'
-    mail_catcher:
-        engine: mailpit
     webdriver:
         engine: chromedriver
+    mail_catcher:
```

Comment:
Why was this moved? If there is a good reason, I recommend adding a comment with an explanation of the "why" it must be at this position.